recipe/build.sh

```diff
@@ -9,20 +13,67 @@ if [[ "${target_platform}" == osx-* ]]; then
   export CXXFLAGS="${CXXFLAGS} -DTARGET_OS_OSX=1"
 fi
 
+bootstrap_build() {
+  echo "Building bootstrap erlang for cross-compilation"
+  # We need to override the compilation toolchain that is setup for cross-compilation,
+  # the bootstrap system needs to be compiled with the guest (x86) toolchain.
+  # Otherwise, configure fails with:
+  # error: Cannot both cross compile and build a bootstrap system
+  local CC CXX CPP LD AR RANLIB
+  CC=${CC_FOR_BUILD}
+  CXX=${CXX_FOR_BUILD}
+  LD=${CC_FOR_BUILD//gnu-cc/gnu-ld}
+  CPP=${CC_FOR_BUILD//gnu-cc/gnu-cpp}
+  AR=${CC_FOR_BUILD//gnu-cc/gnu-ar}
+  RANLIB=${CC_FOR_BUILD//gnu-cc/gnu-ranlib}
```

Comment:
These may be different for macOS as we use a Clang-based toolchain there

Reply:
Indeed, I am just looking at the failed build. :sweat_smile: 
clang also doesn't like empty CFLAGS. I'll try to fix these tomorrow.

Reply:
The build now works! :tada: But it seems we'll have to skip tests in this case.